docs/reference/api/pages.md

```diff
@@ -13,6 +15,18 @@ All [page types](TODO) use a single set of API endpoints, unlike piece types. Di
 |PATCH | [`/api/v1/@apostrophecms/page/:id`](#patch-api-v1-apostrophecms-page-id) | Update only certain fields on a specific page | TRUE |
 |DELETE | Not supported | [See more in PATCH request details.](#moving-pages-to-the-trash) | n/a |
 
+### Additional page endpoints
+
+| Method | Path | Description | Auth required |
+|---------|---------|---------|---------|
+|POST | [`/api/v1/@apostrophecms/page/:id/publish`](#post-api-v1-apostrophecms-page-id-publish) | Publish the draft version of a page | TRUE |
+
+<!-- TODOcument -->
+<!-- |GET | [`/api/v1/@apostrophecms/page/:id?locales=1`](#get-api-v1-apostrophecms-page-id-locales-1) | Request the available locales for a specific page | TRUE |
```

Comment:
We're not writing these up yet, but it took a while to work all this out so I thought I'd leave it for later.